glean_parser/schemas/metrics.1-0-0.schema.yaml

```diff
@@ -294,6 +298,24 @@ definitions:
           - hour
           - day
 
+      memory_unit:
+        title: Memory unit
+        description: |
+          The unit that the incoming memory size values are recorded in.
+
+          The units are the power-of-2 units, so "kilobyte" is pedantically a "kibibyte".
+
+          - kilobyte == 2^10 ==         1,024 bytes
+          - megabyte == 2^20 ==     1,048,576 bytes
+          - gigabyte == 2^30 == 1,073,741,824 bytes
+
```

Comment:
Include a note about truncation/rounding? (Even if there is none)

Reply:
Good point.